presto-main/src/main/java/com/facebook/presto/cost/HistoryBasedPlanStatisticsCalculator.java

```diff
@@ -56,11 +54,7 @@ public PlanNodeStatsEstimate calculateStats(PlanNode node, StatsProvider sourceS
 
     private PlanNode removeGroupReferences(PlanNode planNode, Lookup lookup)
     {
-        if (planNode instanceof GroupReference) {
-            return lookup.resolve(planNode);
-        }
-        List<PlanNode> children = planNode.getSources().stream().map(node -> removeGroupReferences(node, lookup)).collect(toImmutableList());
-        return planNode.replaceChildren(children);
+        return Plans.resolveGroupReferences(planNode, lookup);
```

Comment:
Directly use resolveGroupReferences in the function below, and use import static to directly import the function.

Reply:
done